website/docs/en/config/tools/rspack.mdx

```diff
@@ -235,7 +235,7 @@ export default {
 
 - **Type:** `(rules: RuleSetRule | RuleSetRule[]) => void`
 
-Add additional [Rspack rules](https://rspack.dev/config/module#modulerules).
+Add additional [Rspack rules](https://rspack.dev/config/module#modulerules) to the head of the internal Rspack module rules array.
```

Comment:
We should remind Rspack that the rules are executed in the reverse order of the array.